lumibot/data_sources/tradier_data.py

```diff
@@ -230,6 +238,10 @@ def get_historical_prices(
         if "timestamp" in df.columns:
             df = df.drop(columns=["timestamp"])
 
+        # if type of index is date, convert it to timestamp with timezone info of "America/New_York"
+        if isinstance(df.index[0], date):
+            df.index = pd.to_datetime(df.index, utc=True).tz_convert("America/New_York")
```

Comment:
We should not hard code New York. This is a variable in lumibot so people can change it if they're international